src/main/java/org/jabref/logic/autosaveandbackup/AutosaveManager.java

```diff
@@ -2,12 +2,7 @@
 
 import java.util.HashSet;
 import java.util.Set;
-import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.BlockingQueue;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.RejectedExecutionException;
-import java.util.concurrent.ThreadPoolExecutor;
-import java.util.concurrent.TimeUnit;
+import java.util.concurrent.*;
```

Comment:
We have the convention to never use star-imports (you can also import the code style, see wiki)

Reply:
Thanks, fixed that.